### ISSUE TITLE
Train loss and Val loss were were negative

### ISSUE THREAD

**Issue by kingjames1155** [MIC-DKFZ/nnDetection#107]
Hello All,

### :skull: Bug

I used the nndetection and luna16 datasets for training and found that  Train loss and Val loss were were negative(Almost after the 5th-6th epoch)。I wonder if it's because I only use one-fifth of the Luna 16 dataset

2022-09-06 15:45:56.947 | INFO     | nndet.ptmodule.retinaunet.base:training_epoch_end:252 - Train loss reached: 0.00359
2022-09-06 16:29:43.007 | INFO     | nndet.ptmodule.retinaunet.base:validation_epoch_end:269 - Val loss reached: 0.00189
2022-09-06 16:29:43.130 | INFO     | nndet.ptmodule.retinaunet.base:evaluation_end:284 - mAP@0.1:0.5:0.05: 0.697  AP@0.1: 0.721  AP@0.5: 0.658
2022-09-06 16:29:43.131 | INFO     | nndet.ptmodule.retinaunet.base:evaluation_end:292 - Proxy FG Dice: 0.699
2022-09-06 16:29:43.133 | INFO     | nndet.ptmodule.base_module:validation_epoch_end:88 - This epoch took 85 s
2022-09-06 16:29:43.192 | INFO     | nndet.ptmodule.retinaunet.base:training_epoch_end:252 - Train loss reached: -0.06791
2022-09-06 17:14:03.825 | INFO     | nndet.ptmodule.retinaunet.base:validation_epoch_end:269 - Val loss reached: -0.08439
2022-09-06 17:14:03.932 | INFO     | nndet.ptmodule.retinaunet.base:evaluation_end:284 - mAP@0.1:0.5:0.05: 0.763  AP@0.1: 0.799  AP@0.5: 0.713
2022-09-06 17:14:03.933 | INFO     | nndet.ptmodule.retinaunet.base:evaluation_end:292 - Proxy FG Dice: 0.748
2022-09-06 17:14:03.935 | INFO     | nndet.ptmodule.base_module:validation_epoch_end:88 - This epoch took 81 s
2022-09-06 17:14:03.994 | INFO     | nndet.ptmodule.retinaunet.base:training_epoch_end:252 - Train loss reached: -0.10950
2022-09-06 17:58:59.865 | INFO     | nndet.ptmodule.retinaunet.base:validation_epoch_end:269 - Val loss reached: 0.02581
2022-09-06 17:58:59.972 | INFO     | nndet.ptmodule.retinaunet.base:evaluation_end:284 - mAP@0.1:0.5:0.05: 0.691  AP@0.1: 0.720  AP@0.5: 0.645
2022-09-06 17:58:59.973 | INFO     | nndet.ptmodule.retinaunet.base:evaluation_end:292 - Proxy FG Dice: 0.684
2022-09-06 17:58:59.975 | INFO     | nndet.ptmodule.base_module:validation_epoch_end:88 - This epoch took 91 s
2022-09-06 17:59:00.031 | INFO     | nndet.ptmodule.retinaunet.base:training_epoch_end:252 - Train loss reached: -0.09130
2022-09-06 18:44:10.837 | INFO     | nndet.ptmodule.retinaunet.base:validation_epoch_end:269 - Val loss reached: 0.12469
2022-09-06 18:44:10.938 | INFO     | nndet.ptmodule.retinaunet.base:evaluation_end:284 - mAP@0.1:0.5:0.05: 0.738  AP@0.1: 0.751  AP@0.5: 0.707

### Environment

PyTorch  1.11.0
Python  3.8(ubuntu20.04)
Cuda  11.3
GPU
RTX 3090 

Regards,
Du

**Comment by mibaumgartner** [comment]
Dear @kingjames1155,

It is normal that the loss gets negativ in nnDetection. For some losses (e.g. Dice and GIoU) which are usually defined as '1 - value', we omitted the 1 since it only costs computational resources without changing the gradient.

Best, Michsel

**Comment by kingjames1155** [comment]
> Dear @kingjames1155,
> 
> It is normal that the loss gets negativ in nnDetection. For some losses (e.g. Dice and GIoU) which are usually defined as '1 - value', we omitted the 1 since it only costs computational resources without changing the gradient.
> 
> Best, Michael

Dear Michael

Thank you for your prompt reply. I will try again. Thank you again.

Best,Du

**Comment by kingjames1155** [comment]
Dear Michael
How to load the pre-trained model to continue training

**Comment by mibaumgartner** [comment]
Dear Du,

it is possible to set the mode with the overwrites `[current command] -o train.mode=resume`
